chatwoot/chatwoot
# feat: Show email subject in chat list

# Pull Request Template

## Description
This PR will prepend the email subject before the email content preview in the chat list. 

Fixes https://github.com/chatwoot/chatwoot/issues/7925

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

- Navigate to the account dashboard, and go to the conversation list.
- Ensure that when the automated reply is displayed, the original email subject is visible.
- Ensure that the enhancement doesn't interfere with the readability and formatting of the email thread.
- When a chat thread doesn't have a mail subject, ensure that the chat thread preview doesn't display the hyphen.

## Checklist:
